utils/api.js

```diff
@@ -2,8 +2,10 @@
   API for interacting with the data store
 */
 
+//TODO: Switch to promise-based API
```

Comment:
Definitely want to do this soon. Wrap the fake data below in a promise so 'actions/questions.js' file can handle the API call just like it will in the real App.  When we actually have a backend we can replace it with the real http call.